.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -33,7 +34,14 @@ body:
       label: Logs
       description: Add any logs/error messages to help debug the issue.
     validations:
-      required: true
+      required: false
+  - type: textarea
+    id: screenshots
+    attributes:
+      label: Screenshots
+      description: Upload any screenshots to help debug the issue.
```

Comment:
I noticed there's a lack of alignment on the kind of phrase stored in the `description` fields:

> 1. description : A clear and concise description of what were your expectations.
> 1. description : A clear and concise description of what actually happened.
> 1. description : List the steps to reproduce the issue.
> 1. description: Upload any screenshots to help debug the issue.

a. 1 and 2 are describing the field itself while 3 and 4 are telling the submitter what to do. To align with 3 and 4, we would say something like "Provide a clear and concise description..."
a. 1 needs better alignment with 2 w/r/t to sentence structure. To align better, and assuming we do (a), we would say something like "Provide a clear and concise description of what you expected to happen."

Reply:
Modified @danfowler . Thanks